### ISSUE TITLE
Brain.js problem with importing LSTM network from json

### ISSUE THREAD

**Issue by dszarek04** [BrainJS/brain.js#660]
## _What_ is wrong?

Brain.js throws an error when importing LSTM network from a js file

## _Where_ does it happen?

When trying to run brain.js LSTM network from repl.it

## _How_ do we replicate the issue?

[Stack Overflow link](https://stackoverflow.com/questions/64660493/brain-js-problem-with-importing-lstm-network-from-json)

**Comment by jefftmahoney** [comment]
Hi there ... I think what might be going on is that what you're passing into BrainJS is not actually JSON.

In your code, you have this:
```
fs.readFile('traindata.json', function(err, data) {
  if (err) throw err;
  net.fromJSON(data);
  console.log("file loaded");
});
```

When I tried running your code, I saw the same error you saw.
Using `JSON.parse()` seemed to fix the issue.

In other words, when I changed `net.fromJSON(data);` to `net.fromJSON(JSON.parse(data));`, the error disappeared.
